proximal/examples/test_optical_flow.py

```diff
@@ -0,0 +1,97 @@
+'''
+    Compute the Horn-Schunck optical flow algorithm.
+'''
+import sys
+import numpy as np
+import matplotlib.pyplot as plt
+
+sys.path.append('../..')
+
+import proximal as px
+
+Y, X = np.mgrid[-64:64, -64:64]
+R_squared = 35.0**2
+
+################################################################################
+# Generate images
+
+
+def draw_inverted_cone(X, Y, R_squared, offset=0, shift_edge=False):
+    image = np.sqrt((X + offset)**2 + Y**2)
+    if shift_edge:
+        image[(X + offset)**2 + Y**2 >= R_squared] = 0.0
+    else:
+        image[X**2 + Y**2 >= R_squared] = 0.0
+
+    return np.asfortranarray(image, dtype=np.float32)
+
+
+# Draw a cone at the center
+image1 = draw_inverted_cone(X, Y, R_squared)
+
+# Second image is offset by 0.5 pixel...
+image2 = draw_inverted_cone(
+    X,
+    Y,
+    R_squared,
+    offset=0.5,
+    shift_edge=False,
```

Comment:
Toggle `shift_edge` to True to shift the silhouette of the inverted cone as well. Imitate @jonpry's measurements.